prometheus.tf

```diff
@@ -295,3 +296,63 @@ resource "aws_iam_role_policy" "grafana_datasource" {
   policy = data.aws_iam_policy_document.grafana_datasource.0.json
 }
 
+#########################################################
+# Second Prometheus, only when prometheus_split is true #
+#########################################################
+
+resource "null_resource" "infrastructure" {
```

Comment:
Are you missing a `count = var.split_prometheus ? 1 : 0` here?

Also, the name infrastructure is too vague. How about prometheus_infrastructure?

Reply:
Yes, you are right. Just added